mne_bids/io/__init__.py

```diff
@@ -0,0 +1,15 @@
+"""Initialize IO functions."""
+# Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#          Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#          Teon Brooks <teon.brooks@gmail.com>
+#          Chris Holdgraf <choldgraf@berkeley.edu>
+#          Stefan Appelhoff <stefan.appelhoff@mailbox.org>
+#          Matt Sanderson <matt.sanderson@mq.edu.au>
+#
+# License: BSD (3-clause)
+from mne_bids.io.base import (read_raw_bids, write_raw_bids,
```

Comment:
this is dangerous. You should always use relative imports. Otherwise, you might end up using the wrong files

Reply:
pep8 to my rescue

https://www.python.org/dev/peps/pep-0008/#imports

> Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages)